### ISSUE TITLE
How to use email notification?

### ISSUE THREAD

**Issue by GeorgeUgly** [boredazfcuk/docker-icloudpd#84]
I am using below VARIABLES trying send email notification,

command_line_options="--smtp-host smtp.qq.com --smtp-port 587 --smtp-username xxxxx@qq.com --smtp-password xxxxx" \

during container starting, warning messages show

2021-07-29 23:49:30 WARNING  Additional command line options is depreceated. Please specify all options using the dedicated variables

please help, thanks

**Comment by bauzer714** [comment]
@GeorgeUgly  - AFAIK SMTP is not supported. You should use a supported notification_type and the appropriate variables.

`   --env notification_type=<Choice of Prowl, Pushover, Webhook and Telegram> \`

https://github.com/boredazfcuk/docker-icloudpd#notification-configuration-variables

**Comment by GeorgeUgly** [comment]
@bauzer714 ok, thanks. That would be better if smtp can be added back, it's simple & easy to manage.

**Comment by boredazfcuk** [comment]
I did look into enabling SMTP notifications but decided against it about 18months... I don't remember why, but in some of the early versions of the README.MD it says that it was too much work.

I have a feeling that when I looked into it, it needed an underlying message transfer agent to be able to send the notification e-mails. This would have meant packaging a mail server inside the container, configuring it for the user, and running it as a background service with something like supervisord, It would have added quite a bit of bloat to the container, and I'd already got three notification methods at that point, so it didn't really seem worth it.

```
2021-07-29 23:49:30 WARNING Additional command line options is depreciated. Please specify all options using the dedicated variables
```

This isn't an error, it's just a warning I put in. The script should still pass those commands to the underlying Python app.

Once upon a time, the only variables I was configuring was for authentication and everything else was configured through **command_line_options**.  As time progressed, I added more and more variables until I had everything covered (except for the SMTP stuff). I've left the **command_line_options** variable in there though, in case people had still got their containers configured like that. If I push out an update which kills the **command_line_options** variable, and people's containers upgrade automatically (with something like watch tower), then their containers will stop backing up their photos.

I will remove the ***command_line_options** variable at some point, I just want to give people plenty of notice!
